projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6013,6 +6013,10 @@ public function deprecated_hooks() {
 				'replacement' => null,
 				'version'     => 'jetpack-11.0.0',
 			),
+			'jetpack_dsp_promote_posts_enabled'            => array(
```

Comment:
@jeherve On my test WoA sites I'm seeing:

```
[27-Jan-2023 16:51:07 UTC] PHP Deprecated:  Hook jetpack_dsp_promote_posts_enabled is <strong>deprecated</strong> since version jetpack-11.8.0 with no alternative available. in /wordpress/core/6.1.1/wp-includes/functions.php on line 5768
```

Looks like `jetpack_dsp_promote_posts_enabled` is still referenced in a couple places, will that be able to be cleaned up in the future?

Reply:
The filter is still in use in wpcomsh, indeed. I *think* Calypso still relies on the /me endpoint, and thus that filter, to display some elements. @paulopmt1 would know for sure.

This should happen when 1532-gh-Automattic/dotcom-forge gets addressed. Once that's done, I think we should be safe to remove the filter from wpcomsh.